WtWSMS/localization/english/bookmark/bm_476_l_english.yml

```diff
@@ -2,84 +2,84 @@
  #476 bookmarks
  
  #Little Augustus
- bm_476_little_augustus:0 "Little Augustus"
- bm_476_little_augustus_desc:0 "With the rise of Odovacar, the #italic foederati#! leader who desires the throne of Rome, can anyone save the Western Roman Empire from its collapse?\n\nJulius Nepos, the disgraced general and former Roman emperor of the West, attempts to rally as many men as possible to defeat the #italic foederati#! ruler, whilst throwing all of his weight into seizing the throne in Ravenna once more. Meanwhile, Theodoric of the Ostrogoths looks at the peninsula, weighing his options. Will Rome fall to Odovacar? To Theodoric? Will the young Romulus Augustulus maintain his power against all odds, in the twilight of the Empire? Or will Julius Nepos, a shadow of his former self, rise once more to be hailed as Augustus?\n\nThe Eastern half of the Empire watches the destiny of the West carefully while dealing with its own upheaval."
+ bm_476_little_augustus: "Little Augustus"
+ bm_476_little_augustus_desc: "When Orestes refused the demands of the Skirian and Heruli #italic foederati#! for permanent lands in Italy, they turned to the general Odovacar to lead them in revolt. Odovacar answered their call and now seeks to overthrow the usrper Romulus \"Augustulus\" (meaning \"Little Augustus\"), who is but a mere puppet for his father Orestes.\n\nIn Dalmatia, the legitimate Western Emperor - the one recognized by the East - Julius Nepos, plots to reclaim not just his throne but to restore Western Rome to its rightful place.\n\n Further afield, Theodoric of the Ostrogoths looks on, carefully considering his options.\n\nWill the young Romulus, guided by the traitor Orestes, somehow manage to salvage the situation? Will Julius be able to reclaim his throne? Will Odovacar manage to provide his people with the lands and power they desire? Or will something else happen?"
```

Comment:
I'm not opposed to changing the description but I'm not convinced by this wording. You make a value statements when claiming Nepos is legitimate because recognised by the ERE, one which isn't recognised in game. Secondly there is nothing which proves us Nepos had plans beyond Italy. Then you removed the tension around Theodoric, while we don't need to orientate towards Italy the way you write it doesn't justify his mention, all rulers consider their options.  And Orestes being considered a traitor is also not really neutral and not reflected in game. There is also a typo in "usurper" which I suggest removing altogether.

> In Dalmatia, the disgraced general and former Western Emperor Julius Nepos, is drawing up militarily plans to seize the throne in Ravenna by force, but has kept many powerful ennemies. 
> Further afield, the former Eastern foederati Theodoric of the Ostrogoths looks on, carefully considering his options in facing the power of Constantinople and the rival Thracian Goths
> Will the young Romulus, guided by his ambitious father Orestes, somehow manage to salvage the situation?
> Will Nepos be able to leave Dalmatia and rise once more to be hailed as Augustus ?

Reply:
Good points; implemented.